openmdao/components/meta_model_structured.py

```diff
@@ -85,12 +49,56 @@ def _interp_methods():
 
     @staticmethod
     def methods():
-        """Return a list of valid interpolation method names."""
+        """
+        Return a list of valid interpolation method names.
+
+        Returns
+        -------
+        list
+            Valid interpolation name strings.
+        """
         return ['slinear', 'cubic', 'quintic']
 
     def __init__(self, points, values, method="slinear", bounds_error=True,
                  fill_value=np.nan, spline_dim_error=True):
-        """Initialize instance of interpolation class."""
+        """
+        Initialize instance of interpolation class.
+
+        Parameters
+        ----------
+        points : tuple of ndarray of float, with shapes (m1, ), ..., (mn, )
+            The points defining the regular grid in n dimensions.
+
```

Comment:
we're not consistent on our line spacing for parameters.  In most of our existing code we don't put a blank line between each of our parameter descriptions.  I think we should stick with that since our docstrings are long enough as it is.

Reply:
Agreed.  Let me see how many of these need fixing.

Reply:
Looks like a lot of these to fix...will have them in soon.

Reply:
OK, fixed like 150 of these spacing inconsistencies.